contribs/gmf/src/authentication/component.html

```diff
@@ -36,14 +32,17 @@
   </form>
 </div>
 
+<div class="alert alert-warning" ng-show="$ctrl.infoMessage">
+  <span>{{ $ctrl.infoMessage }}</span>
+</div>
+
```

Comment:
Move it under username to not have it between username info and intranet info (more coherent)